src/test/java/com/sngular/kloadgen/testutil/WireMockLoaderUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.sngular.kloadgen.testutil;
+
+import static com.github.tomakehurst.wiremock.common.Json.read;
+
+import java.io.IOException;
+
+import com.github.tomakehurst.wiremock.junit5.WireMockRuntimeInfo;
+import com.github.tomakehurst.wiremock.stubbing.StubMappingCollection;
+
+public final class WireMockLoaderUtil {
+
+    private final FileHelper fileHelper = new FileHelper();
+
+    public void setUpMocks(final WireMockRuntimeInfo wmRuntimeInfo) throws IOException {
```

Comment:
Why is this needed? As I recall the default behavior is get all the json files on that folder. It was failing somehow?